src/recipe/test/unit/recipe.service.spec.ts

```diff
@@ -86,20 +99,47 @@ describe('Recipe Service', () => {
     expect(result.ingredients[0].amount).toBe('spoon');
   });
 
-  it('should find one recipe by id', async () => {
+  it('should return cached data if available', async () => {
     const queryBusExecuteFindOne = jest.spyOn(queryBus, 'execute');
+    const cacheServiceGet = jest
+      .spyOn(cacheService, 'get')
+      .mockResolvedValue(recipe);
+
+    const recipeFromDb = await recipeService.createRecipe(recipe, testUser.id);
+    const result = await recipeService.findRecipeById(recipeFromDb.id);
+
+    expect(result.name).toBe('Dumplings');
```

Comment:
if this test case is only about checking if the get method from cacheService was called there is no need to test all properties on the response that you have mocked

Reply:
also while writing tests you should try to test only one part of logic. Right now, the description of this test case indicates that you are testing if the correct method from cacheService is called.